src/SpriteSheet.js

```diff
@@ -1,4 +1,5 @@
 import { Animated, Easing, View } from "react-native";
+import Svg,{Image}from 'react-native-svg';
```

Comment:
What's this for?

Reply:
must've missed this when i was testing a different approach to making images not pixelated when enlarging. will remove.